Sources/Kitura-CredentialsTwitter/CredentialsTwitter.swift

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright Jacob Van Order 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+import Kitura
+import KituraNet
+import LoggerAPI
+import Credentials
+
+import SwiftyJSON
+
+import Foundation
+
+let oAuthTokenKey = "oauth_token"
+let oAuthTokenSecretKey = "oauth_token_secret"
+let oAuthVerifierKey = "oauth_verifier"
+let userIDKey = "user_id"
+let screenNameKey = "screen_name"
```

Comment:
I'd argue that these should be inside the scope of the class.  Then they don't hang around in memory all the time.